modules/plugins/jboss-as-7/src/main/java/org/rhq/modules/plugins/jbossas7/JBossProductType.java

```diff
@@ -143,6 +144,8 @@ private static JBossProductType determineJBossProductTypeViaProductConfFile(File
                 productType = JBossProductType.BPMS;
             } else if (slot.equals("dv")) {
                 productType = JBossProductType.JDV;
+            } else if (slot.equals("wildfly-full")) {
```

Comment:
Wildfly 7,8,9,10 (7=as7) can be detected by management-major-version iirc

wf 9 has management-major-version=3
wf 10 has management-major-version=4
as7 has 1 and I think wf8 had 2 (needs to be confirmed).

Also 

on wf 10
release-version=2.0.10.Final

wf9(.01)
release-version=1.0.1.Final

and so on, but I believe the management-major-version
is the most unique per major WF version.